modules/photons_control/clean.py

```diff
@@ -0,0 +1,216 @@
+from photons_messages import LightMessages, LightLastHevCycleResult
+from photons_app.tasks import task_register as task
```

Comment:
nit: I put imports into groups.

so here'd it'd be

```
from photons_control.script import FromGenerator
from photons_control.planner import Skip

from photons_app.tasks import task_register as task

from photons_messages import LightMessages, LightLastHevCycleResult

from delfick_project.norms import sb, Meta
from datetime import timedelta

try:
    import humanize
except ImportError:
    humanize = None
```

cause it's 

```
<current module>

<photons_app imports>

<other photons imports>

<non photons imports>
```

Reply:
You know, I thought I did this. I shall fix.